### PR TITLE
add Styling-sanjit

### PR DESCRIPTION
**Pull request by sanjitcodes**
For #50

### CODE REVIEW

File: participants/sanjitcodes.md [hunk 1]
```diff
@@ -2,4 +2,19 @@ Name: SANJIT ANAND
...
+
+# **HOBBIES**
+1. Lawn Tennis
+2. Cricket 
+3. Piano
+4. Learning New and Exciting Things
+
+# **EDUCATION**
+
+Class 1-12 -- Essar International School
```
Comment (lisa761) [inline participants/sanjitcodes.md:17]: I think whatever you were trying to do with `--` didn't work? You can look at your current md file [here](https://github.com/ACM-IGDTUW/Open-Source-Challenge/blob/4262b6bc17d9ba5c4c769fdd36ea6a24042f8d46/participants/sanjitcodes.md).

**Comment by sanjitcodes** [comment]
Should I make another PR for modifying the changes ?

**Comment by soumyaa1804** [comment]
You can make changes to this branch only. Make the changes and then add, commit and push the changes. Make sure you are on the same branch.

**Comment by soumyaa1804** [comment]
Remember to leave a line to give a line break.

**Comment by sanjitcodes** [comment]
Done changes, you may merge it

**Comment by sanjitcodes** [comment]
So I have to make another one ?

File: participants/sanjitcodes.md [hunk 2]
```diff
@@ -1,5 +1,26 @@
...
+
+##GitHub username: sanjitcodes
+
+##University: NIT, Surat
+
+##Year: 1st
+
+##Branch: Electronics and Communication Engineering
+
+### I am from Amritsar, Punjab.
```
Comment (soumyaa1804) [inline participants/sanjitcodes.md:12]: @sanjitcodes nicely done! Just one more change and you are good to go. You need to add space between hashes and name, GitHub name, university, year and branch for markdown to consider it as H2 syntax

**Comment by sanjitcodes** [comment]
>> is it okay ?

**Pull request merged**